src/quicer.erl

```diff
@@ -468,25 +468,30 @@ start_stream(Conn, Opts) when is_map(Opts) ->
   quicer_nif:start_stream(Conn, maps:merge(default_stream_opts(), Opts)).
 
 %% @doc Send binary data over stream, blocking until send request is handled by the transport worker.
+%% either successed or cancelled
 -spec send(stream_handler(), iodata()) ->
         {ok, BytesSent :: pos_integer()}          |
+        {error, cancelled}                        |
         {error, badarg | not_enough_mem | closed} |
         {error, stream_send_error, atom_reason()}.
 send(Stream, Data) ->
   case quicer_nif:send(Stream, Data, _IsSync = 1) of
     %% @todo make ref
     {ok, _Len} = OK ->
       receive
-        {quic, send_completed, Stream, _} ->
-          OK
+        {quic, send_completed, Stream, ?QUIC_SEND_COMPLETE_SUCCESS} ->
+          OK;
+        {quic, send_completed, Stream, ?QUIC_SEND_COMPLETE_SUCCESS} ->
```

Comment:
Duplicated clause?

Reply:
Nice catch, was testing that and forget to rollback....